Trackovid19-web/src/app/helpers/local-storage.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+
+export enum LoginMethod {
+  FACEBOOK = 'facebook',
+}
+
+@Injectable({ providedIn: 'root' })
+export class LocalStorageHelper {
+  setLoginMethod(loginMethod: LoginMethod) {
+    localStorage.setItem('loginmethod', loginMethod);
+  }
+
+  getLoginMethod(): LoginMethod {
+    const method = localStorage.getItem('loginmethod');
+    if (method) {
+      return LoginMethod[method.toUpperCase()];
+    }
+    return undefined;
+  }
+
+  setShareStatus(shareStatus: string) {
+    localStorage.setItem('share', shareStatus);
+  }
+
+  getShareStatus(): string {
+    return localStorage.getItem('share');
+  }
+
+  setGDPR(gdpr: string) {
+    localStorage.setItem('gdpr', gdpr);
+  }
+
+  getGPRD(): string {
```

Comment:
`setGDPR` and `getGPRD` have different acronyms

Reply:
Done!